.github/workflows/coverage.yml

```diff
@@ -17,6 +17,12 @@ jobs:
   coverage:
     runs-on: ubuntu-latest
 
+    strategy:
+      matrix:
+        go-version:
+          - '1.21'
+          - '1.22'
```

Comment:
Thanks! 
Should we add this in the other workflows too, or do you think coverage is enough since this will run unit tests?

Reply:
i think only here is enough, the CI one is checking the lint; maybe we upgrade that to 1.22 directly

Reply:
I realised that the coverage report for 1.22 will override that of 1.21, but that's fine since codecov only expects one version of the coverage output for a PR.